float64[6] joints_bottom_up_m
float64[6] joints_bottom_up_f
float64[6] joints_top_down_m
float64[6] joints_top_down_f
